### ISSUE TITLE
send email alert for reorder level

### ISSUE THREAD

**Issue by walterhumphreys** [jekkos/opensourcepos#495]
hi guy can someone give me a hint , i need that reorder level alert goes to an email address i can create a template with php and get the info from the database but how to send it?

**Comment by daN4cat** [comment]
To see how to send an email check the send receipt or invoice over email.
Unless you mean how to have something automated to send you that email and how to do that?

**Comment by walterhumphreys** [comment]
Yes automated email example to my email. Reorder level is 90 once is 89 i will like to recieve an email notification about that with the item details

**Comment by daN4cat** [comment]
You probably need some functionality triggered by a cron task that runs in the background and does a check and send an email.

You can check on github if there is any example with CodeIgniter, maybe this one: https://github.com/bianchins/crono could give some hint about how to handle timed execution?

**Comment by rkaombe** [comment]
This option need to be embedded into the system, maybe in future updates they will add this feature. I'm also looking forward to this
